editor/src/components/canvas/canvas-strategies/strategies/grid-move-absolute.ts

```diff
@@ -285,6 +335,19 @@ function runGridMoveAbsolute(
     return []
   }
 
+  const coordinateSystemBounds =
```

Comment:
Unless I'm mistaken I think this could be passed to the `getNewGridElementPropsCheckingOriginalGrid` function here, and then it would also make sense to check for it to be null here at line `340` instead of `?? zeroCanvasRect` since `getNewGridElementPropsCheckingOriginalGrid` would return `null` anyways and do nothing on line `370`.

Reply:
Yeah, that's cleaner, change made in b2004f0b8.